### ISSUE TITLE
Causes process to crash

### ISSUE THREAD

**Issue by foguetes** [jeremyhart/QuickLook.Plugin.HelixViewer#5]
After installing this plugin quicklook imediately stops running when I attempt to preview any file or access quicklook via the system tray.

**Comment by Darthagnon** [comment]
I've got this problem, too. I'm using the MSI version of Quicklook, installed to C:/Windows Tweaks/ and I'm running it on a laptop with:

 - Touchscreen
 - 1TB Mechanical HDD (i.e. no SSD) with plenty of free space
 - Win10 v1709
 - 8GB RAM
 - Intel Core i7-4500U

I have seen the plugin working, when I dropped it into a portable installation of Quicklook on a more powerful university PC on Win10 v1803, and it worked brilliantly then, but on this home laptop, it just causes Quicklook to crash and exit without any error messages or anything.

Hopefully it's fixable, as the plugin, when it works, is amazing!
